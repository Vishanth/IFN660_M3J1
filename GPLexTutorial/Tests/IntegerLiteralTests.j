1_0_5L
0L
9002L
10l
023_32l