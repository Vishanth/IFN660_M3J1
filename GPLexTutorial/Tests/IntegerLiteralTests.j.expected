INTEGER LITERAL (1_0_5L)
INTEGER LITERAL (0L)
INTEGER LITERAL (9002L)
INTEGER LITERAL (10l)
INTEGER LITERAL (023_32l)
EOF